### ISSUE TITLE
Animated SVGs causing 50-100% load on 1-2 CPU cores

### ISSUE THREAD

**Issue by c2h5oh** [drone/drone-ui#315]
tl;dr; Opening drone tab with a pending/in progress job makes my laptop CPU temperature increase by 40 degrees within 1.5 minutes

SVG animations are not GPU accelerated and their performance haven't been given much attention by browser developers. 
The tiny animations indicating a pending / in progress job are enough to have 1-2 cores running at 50-100% in both Chrome 78 and Firefox 70.

Those animations are fixed size right now, so an animated gif would look exactly as crisp as svg and negligible amount to page size (2kb maybe?).

https://discourse.drone.io/t/drone-ui-very-high-cpu-usage-because-of-animated-svg-spinner/6110/2

**Comment by c2h5oh** [comment]
bump?

**Comment by xoxys** [comment]
This problem definitely still exists in the current Firefox on Linux... 

![image](https://user-images.githubusercontent.com/3391958/142424650-9362d200-3caf-4b5b-b06f-34834ec68b69.png)
